skt/executable.py

```diff
@@ -422,6 +427,12 @@ def setup_parser():
                               help="Path to kernel config to use")
     parser_build.add_argument("--cfgtype", type=str, help="How to process "
                               "default config (default: olddefconfig)")
+    parser_build.add_argument(
+        "--enable-debuginfo",
+        type=bool,
+        default=False,
+        help="build kernel with debuginfo (default: disabled)"
```

Comment:
Could you please start the help text with a capital letter to match the rest?

Reply:
Sure.